project.clj

```diff
@@ -34,7 +34,7 @@
 
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [cheshire "5.5.0"]
-                 [org.clojure/core.match "0.2.2"]
+                 [org.clojure/core.match "0.3.0-alpha4" :exclusions [org.clojure/tools.analyzer.jvm]]
```

Comment:
... sorry, I misread this. I needed a new version of core.async, which needs a newer tools.analyzer.jvm. Had to pull in a newer core.match as well to make that dep work.